pkg/deviceplugin/server.go

```diff
@@ -149,6 +152,10 @@ func (srv *server) Allocate(ctx context.Context, rqt *pluginapi.AllocateRequest)
 }
 
 func (srv *server) PreStartContainer(ctx context.Context, rqt *pluginapi.PreStartContainerRequest) (*pluginapi.PreStartContainerResponse, error) {
+	if srv.preStartContainer != nil {
+		return new(pluginapi.PreStartContainerResponse), srv.preStartContainer(rqt)
+	}
+
 	return nil, errors.New("PreStartContainer() should not be called")
```

Comment:
the error message could be updated since we now have an implementation for this?